doc/man3/PEM_read_bio_PrivateKey.pod

```diff
@@ -170,8 +170,8 @@ EVP_PKEY structure. The write routines use PKCS#8 private key format and are
 equivalent to PEM_write_bio_PKCS8PrivateKey().The read functions transparently
 handle traditional and PKCS#8 format encrypted and unencrypted keys.
 
-PEM_write_bio_PrivateKey_traditional() writes out a private key in legacy
-"traditional" format.
+PEM_write_bio_PrivateKey_traditional() writes out a private key in the
+"traditional" format with a simple private key marker.
```

Comment:
To me the word "legacy" here was implying that this format shouldn't normally be used unless you need it. That doesn't come across in the new wording.